controllers/grid/users/reviewer/ReviewerGridRow.inc.php

```diff
@@ -123,7 +145,7 @@ function initialize($request, $template = null) {
 			$canCurrentUserGossip = ServicesContainer::instance()
 				->get('user')
 				->canCurrentUserGossip($reviewAssignment->getReviewerId());
-			if ($canCurrentUserGossip) {
+			if ($canCurrentUserGossip && (!$this->_isCurrentUserAssignedAuthor || !$isAuthorBlind)) {
```

Comment:
if it is not an author: doesn't the check `$canCurrentUserGossip` already clears the access?
thus it could maybe only be `if ($canCurrentUserGossip && !$isAuthorBlind)` ?

Reply:
Good catch!